src/AccessControl/libraries/LibAccessControl.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.30;
+
+library LibAccessControl {
+
+    /// @notice Emitted when the admin role for a role is changed.
+    /// @param role The role that was changed.
+    /// @param previousAdminRole The previous admin role.
+    /// @param newAdminRole The new admin role.
+    event RoleAdminChanged(bytes32 indexed role, bytes32 indexed previousAdminRole, bytes32 indexed newAdminRole);

```

Comment:
All parameter names to events, errors, functions need to be prefixed with "_".  For example `_role`